docs/content/policy-language.md

```diff
@@ -3802,6 +3807,10 @@ Unused imports | Unused [imports](../policy-language/#imports) are prohibited.
 `input` and `data` reserved keywords | `input` and `data` are reserved keywords, and may not be used as names for rules and variable assignment.                                                                                                                                                      | 1.0
 Use of deprecated built-ins | Use of deprecated functions is prohibited, and these will be removed in OPA 1.0. Deprecated built-in functions: `any`, `all`, `re_match`,  `net.cidr_overlap`, `set_diff`, `cast_array`, `cast_set`, `cast_string`, `cast_boolean`, `cast_null`, `cast_object` | 1.0
 
+{{< info >}}
+If the `rego.v1` import is present in a module, all strict mode checks documented above expect the unused local assignment and unused imports checks are enforced on the module.
+{{< /info >}}
```

Comment:
Great 👍 .

Should we also mention that this import will also enforce the use of the `if` keyword for rules with a body, and the `contains` keyword for partial-set rules?

Reply:
Added a note about that.